### PR TITLE
IBX-6500: Page Builder product guide added in developer documentation

### PR DESCRIPTION
**Pull request by julitafalcondusza**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | (https://issues.ibexa.co/browse/IBX-6500)
| Versions      | master

Page Builder product guide added in developer documentation.

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/content_management/pages/page_builder_guide.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+
+### Create page
+
+To create a new page:
+
+1\. In the upper toolbar click **Content**.
+
+2\. Select **Content structure**.
+
+3\. On the right-side toolbar, click **Create content** and from the list of Content items select Landing Page.
```
Comment (MagdalenaZuba) [inline docs/content_management/pages/page_builder_guide.md:22]:
```suggestion
3\. On the right-side toolbar, click **Create content** and from the list of Content items select **Landing Page**.
```

File: docs/content_management/pages/page_builder_guide.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+
+4\. Select the layout and click **Create**.
+
+![Create page](create_page.png)
+
+You can also add a new page in the Page Builder. Check 'Create page in Page Builder' section below.
```
Comment (MagdalenaZuba) [inline docs/content_management/pages/page_builder_guide.md:28]:
```suggestion
You can also add a new page in the Page Builder. Check [Create page in Page Builder](#create-page-in-page-builder) section below.
```

File: docs/content_management/pages/page_builder_guide.md [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+
+|Option|Description|
+|------|-----------|
+|Close|Close the page without saving it.|
+|Send to review|Save the page and send it to review.|
+|Publish / Publish later|Publish the page or schedule publishing for later.|
+|Save draft|Save the page draft*.|
+|Delete draft|Delete the page draft.|
+
+*To help you preserve your work, system saves drafts of Content items automatically. For more information, see [Autosave](https://doc.ibexa.co/projects/userguide/en/master/content_management/content_versions/#autosave).
```
Comment (MagdalenaZuba) [inline docs/content_management/pages/page_builder_guide.md:88]:
```suggestion
*To help you preserve your work, system saves drafts of Content items automatically. For more information, see [Autosave]([[= user_doc =]]/content_management/content_versions/#autosave).
```

File: docs/content_management/pages/page_builder_guide.md [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+
+Page Builder has two main views that you can use while creating a page:
+
+- Elements toolbar - consists of all available elements that you can use by dragging them and dropping on a drop zone.
+
+![Elements toolbar](elements_toolbar.png)
+
+- Structure view - shows a structure of the page, including its division into zones and the blocks that it contains. It follows the behavior of the content tree. Structure view has ability to reorder blocks using drag and drop.
```
Comment (MagdalenaZuba) [inline docs/content_management/pages/page_builder_guide.md:96]:
```suggestion
- Structure view - shows a structure of the page, including its division into zones and the blocks that it contains. It follows the behavior of the Content Tree. Structure view has the ability to reorder blocks using drag and drop.
```

File: docs/content_management/pages/page_builder_guide.md [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+
+![Structure view](structure_view.png)
+
+#### Create page in Page Builder
+
+To open the Page Builder, click **Site** and select a website from the list that appears next to the List icon.
+On the Page Builder toolbar click the plus button and from the list select Landing Page.
```
Comment (MagdalenaZuba) [inline docs/content_management/pages/page_builder_guide.md:103]:
```suggestion
On the Page Builder toolbar click the plus button and from the list select **Landing Page**.
```

File: docs/content_management/pages/page_builder_guide.md [hunk 6]
```diff
@@ -0,0 +1,213 @@
...
+
+##### Choose layout
+
+For newly created Page you can choose a [layout](https://doc.ibexa.co/projects/userguide/en/latest/content_management/configure_ct_field_settings/#available-page-layouts) which defines the available zones.
```
Comment (MagdalenaZuba) [inline docs/content_management/pages/page_builder_guide.md:107]:
```suggestion
For newly created Page you can choose a [layout]([[= user_doc =]]/content_management/configure_ct_field_settings/#available-page-layouts) which defines the available zones.
```

File: docs/content_management/pages/page_builder_guide.md [hunk 7]
```diff
@@ -0,0 +1,213 @@
...
+Applying a layout divides the Page into the defined zones. The zones are placeholders for Content items.
+
+On the Page creation modal, select the layout and click **Create draft**.
+Now you are ready to add blocks of content to the Page.
+
+The page layouts that an editor has access to are up to you to choose.
+In the `Select layouts` section, you can select layouts that you want to be available for the Page.
+
+The default, built-in Page layout has only one zone, but developers can create other layouts in configuration.
+For more information, see [Configure layout](https://doc.ibexa.co/en/latest/templating/render_content/render_page/#configure-layout).
```
Comment (MagdalenaZuba) [inline docs/content_management/pages/page_builder_guide.md:120]:
```suggestion
For more information, see [Configure layout](render_page.md#configure-layout).
```

File: docs/content_management/pages/page_builder_guide.md [hunk 8]
```diff
@@ -0,0 +1,213 @@
...
+- Move up - allows you to change position of the block on the page by moving it up
+- Move down - allows you to change position of the block on the page by moving it down
+- Configuration - allows you to access configuration window
+- Duplicate - duplicates existing block by creating its copy
+- Refresh - refreshes preview of the block
+- Delete - deletes existing block
+
+#### Schedule content
+
+Page Builder comes with a Scheduler - is a tool that allows you to schedule content publication.
```
Comment (adriendupuis) [inline docs/content_management/pages/page_builder_guide.md:161]: This one verb "is" sounds weird here as not in a definition list.
I would also avoid "publication" that could mislead to a confusion with "publish" concept.
```suggestion
Page Builder comes with a Scheduler - a tool that allows you to schedule content appearance.
```

File: docs/content_management/pages/page_builder_guide.md [hunk 9]
```diff
@@ -0,0 +1,213 @@
...
+To customize your page in Page Builder you need to add blocks.
+To do it, access Elements toolbar, drag page block that you want to use and drop it on the empty place on a drop zone.
+
+When you add a new block to the drop zone, before you drop it, a blue line appears  - it helps you see the position of the newly added block in relation to other, already added blocks.
+
+![Drop zone line](drop_zone_line.png)
+
+Ready-to-use blocks available in [[= product_name =]] have their own, unique functions, but you can also [add your own, custom blocks](create_custom_page_block.md). All available tools and settings, that Page Builder comes with, enable you to customize the content appearing on the page.
+
+You can check all ready-to-use blocks available in Page Builder in User Documentation, [Block reference page](https://doc.ibexa.co/projects/userguide/en/latest/content_management/block_reference/).
```
Comment (MagdalenaZuba) [inline docs/content_management/pages/page_builder_guide.md:133]:
```suggestion
You can check all ready-to-use blocks available in Page Builder in User Documentation, [Block reference page]([[= user_doc =]]/content_management/block_reference/).
```

File: docs/content_management/pages/page_builder_guide.md [hunk 10]
```diff
@@ -0,0 +1,213 @@
...
+
+#### Work with blocks
+
+Working with blocks is intuitive. You don't have to worry about placing blocks in the proper place from the start - you can reorder the at any time.
```
Comment (MagdalenaZuba) [inline docs/content_management/pages/page_builder_guide.md:137]:
```suggestion
Working with blocks is intuitive. You don't have to worry about placing blocks in the proper place from the start - you can reorder them at any time.
```

File: docs/content_management/pages/page_builder_guide.md [hunk 11]
```diff
@@ -0,0 +1,213 @@
...
+You can manage each block by accessing its settings. To do it, click settings icon next to the block's name.
+
+![Block settings](block_settings.png)
+
+Available settings are:
+
+- Move up - allows you to change position of the block on the page by moving it up
+- Move down - allows you to change position of the block on the page by moving it down
+- Configuration - allows you to access configuration window
+- Duplicate - duplicates existing block by creating its copy
```
Comment (MagdalenaZuba) [inline docs/content_management/pages/page_builder_guide.md:155]: blank copy or with settings?

File: docs/content_management/pages/page_builder_guide.md (hunk 8 above)
Comment (MagdalenaZuba) [inline docs/content_management/pages/page_builder_guide.md:161]:
```suggestion
Page Builder comes with a Scheduler, it allows you to schedule content publication.
```

File: docs/content_management/pages/page_builder_guide.md [hunk 12]
```diff
@@ -0,0 +1,213 @@
...
+
+- **Scheduler tab** - it's available in the configuration of all Page blocks. In this tab you can set the date and time when the block becomes visible and when it disappears from a Page.
+
+![Scheduler tab](scheduler_tab.png)
+
+- **Content Scheduler** - it's one of the block available in Page Builder Elements menu. To proceed with the schedule, go to **Basic** tab of the block, then click **Select contennt** and confirm your choice, and set date and time in the Content airtime settings window.
+
+![Content Scheduler](content_scheduler.png)
+
+For more information, see [Schedule publication](https://doc.ibexa.co/projects/userguide/en/latest/content_management/schedule_publishing/).
```
Comment (MagdalenaZuba) [inline docs/content_management/pages/page_builder_guide.md:173]:
```suggestion
For more information, see [Schedule publication]([[= user_doc =]]/content_management/schedule_publishing/).
```

File: docs/content_management/pages/page_builder_guide.md [hunk 13]
```diff
@@ -0,0 +1,213 @@
...
+## Benefits
+
+### Manage your pages without technical skills
+
+Thanks to intuitive and plain Page Builder interface, you can create and manage your website without the need of having advanced technical skills.
+Drag-and-drop elements toolbar, separated page zones and transparent sections - these are the elements that make working with Page Builder really intuitive and quick.
+
+### Self schedule content, special offers and campaigns
+
+One of the most important tool that Page Builder offers, is a Scheduler. It allows you to set and schedule specific date and time, when content has to be published. In result, you can plan timeline of your new publications, without the need of manual publishing each of them.
```
Comment (MagdalenaZuba) [inline docs/content_management/pages/page_builder_guide.md:184]:
```suggestion
One of the most important tool that Page Builder offers, is a Scheduler. It allows you to set and schedule specific date and time, when content has to be published or hidden. In result, you can manage timeline of publications, without the need of manual publishing or hiding each of them.
```

File: docs/content_management/pages/page_builder_guide.md [hunk 14]
```diff
@@ -0,0 +1,213 @@
...
+### Create high-converting and fully-targeted landing pages
+
+Page Builder allows you to create highly customizable websites. You can build modifiable and targeted landing pages, that meet your needs.
+Each dynamic blocks has its own settings, properties and design that you can set up in your way to customize the content appearing on the page.
+Additionaly, if you feel comfortable with your technical skills, you can configure your own elements, for example, new customized layout, or block.
+
+### Increase sales with highly personalized campaigns
+
+Personalized campaigns are one of the factors that can increase your sales. With Page Builder you can achieve it, by using customization and time Scheduler.
+Anytime you can edit your page and change the position of specific block to make it more visible. What's more, Page Builder offers you ready-to-use page blocks that can help to create content tailored to each individual customer:
```
Comment (MagdalenaZuba) [inline docs/content_management/pages/page_builder_guide.md:195]:
```suggestion
Anytime you can edit your page and change the position of specific block to enhance visibility. Additionaly, Page Builder offers you a selection of ready-to-use page blocks that can help you to create content tailored to each individual customer:
```

**Review by MagdalenaZuba — CHANGES_REQUESTED** [review]

File: docs/content_management/pages/page_builder_guide.md [hunk 15]
```diff
@@ -0,0 +1,213 @@
...
+4\. Select the layout and click **Create**.
+
+![Create page](create_page.png)
+
+You can also add a new page in the Page Builder. Check 'Create page in Page Builder' section below.
+
+### Edit page
+
+You can edit any existing page by using Page Builder. To do it, in the Back Office go to **Content** and select **Content structure**.
+Then, from the Content Tree choose the page that you want to edit and click **Edit**.
```
Comment (juskora) [inline docs/content_management/pages/page_builder_guide.md:33]:
```suggestion
Then, from the Content Tree choose the page and click **Edit**.
```

File: docs/content_management/pages/page_builder_guide.md [hunk 16]
```diff
@@ -0,0 +1,213 @@
...
+
+The default, built-in Page layout has only one zone, but developers can create other layouts in configuration.
+For more information, see [Configure layout](https://doc.ibexa.co/en/latest/templating/render_content/render_page/#configure-layout).
+
+#### Add blocks
+
+To customize your page in Page Builder you need to add blocks.
+To do it, access Elements toolbar, drag page block that you want to use and drop it on the empty place on a drop zone.
+
+When you add a new block to the drop zone, before you drop it, a blue line appears  - it helps you see the position of the newly added block in relation to other, already added blocks.
```
Comment (juskora) [inline docs/content_management/pages/page_builder_guide.md:127]: Or maybe: drop it in the blue highlighted area? Instead of blue lines.

**Review by juskora — CHANGES_REQUESTED** [review]

File: docs/content_management/pages/page_builder_guide.md [hunk 17]
```diff
@@ -0,0 +1,213 @@
...
+![Content Scheduler](content_scheduler.png)
+
+For more information, see [Schedule publication](https://doc.ibexa.co/projects/userguide/en/latest/content_management/schedule_publishing/).
+
+## Benefits
+
+### Manage your pages without technical skills
+
+Thanks to intuitive and plain Page Builder interface, you can create and manage your website without the need of having advanced technical skills.
+Drag-and-drop elements toolbar, separated page zones and transparent sections - these are the elements that make working with Page Builder really intuitive and quick.
```
Comment (dabrt) [inline docs/content_management/pages/page_builder_guide.md:180]:
```suggestion
Drag-and-drop elements toolbar, separated page zones and transparent sections - these are the elements that make working with Page Builder really intuitive and quick.
```
Drag-and-drop elements toolbar - sounds a lot like a command. Maybe just "The **Elements** toolbar"
"separated" - as in "well defined" or "visible"?
"transparent sections" - in what sense? What do you mean by a section here?

File: docs/content_management/pages/page_builder_guide.md (hunk 13 above)
Comment (dabrt) [inline docs/content_management/pages/page_builder_guide.md:184]:
```suggestion
One of the most important tools that Page Builder offers, is a Scheduler. It allows you to set and schedule a specific date and time for the content to be published. As a result, you can create a timeline of your new publications, without having to publish each of them manually.
```

File: docs/content_management/pages/page_builder_guide.md [hunk 18]
```diff
@@ -0,0 +1,213 @@
...
+
+### Self schedule content, special offers and campaigns
+
+One of the most important tool that Page Builder offers, is a Scheduler. It allows you to set and schedule specific date and time, when content has to be published. In result, you can plan timeline of your new publications, without the need of manual publishing each of them.
+
+### Create high-converting and fully-targeted landing pages
+
+Page Builder allows you to create highly customizable websites. You can build modifiable and targeted landing pages, that meet your needs.
```
Comment (dabrt) [inline docs/content_management/pages/page_builder_guide.md:188]:
```suggestion
Page Builder allows you to create highly customizable websites. You can build modifiable and targeted landing pages that meet your needs.
```

File: docs/content_management/pages/page_builder_guide.md [hunk 19]
```diff
@@ -0,0 +1,213 @@
...
+Each dynamic blocks has its own settings, properties and design that you can set up in your way to customize the content appearing on the page.
+Additionaly, if you feel comfortable with your technical skills, you can configure your own elements, for example, new customized layout, or block.
```
Comment (dabrt) [inline docs/content_management/pages/page_builder_guide.md:190]:
```suggestion
Additionaly, if you feel comfortable with your technical skills, you can configure your own elements, for example, a new customized layout, or block.
```

File: docs/content_management/pages/page_builder_guide.md [hunk 20]
```diff
@@ -0,0 +1,213 @@
...
+
+### Increase sales with highly personalized campaigns
+
+Personalized campaigns are one of the factors that can increase your sales. With Page Builder you can achieve it, by using customization and time Scheduler.
```
Comment (dabrt) [inline docs/content_management/pages/page_builder_guide.md:194]:
```suggestion
Personalized campaigns are one of the factors that can increase your sales. 
With Page Builder, you can achieve it by using customization and time Scheduler.
```

File: docs/content_management/pages/page_builder_guide.md (hunk 14 above)
Comment (dabrt) [inline docs/content_management/pages/page_builder_guide.md:195]:
```suggestion
Anytime you can edit your page and change a position of a block to make it more visible. 
What's more, Page Builder offers you ready-to-use page blocks that can help create content tailored to each individual customer:
```

File: docs/content_management/pages/page_builder_guide.md [hunk 21]
```diff
@@ -0,0 +1,213 @@
...
+Anytime you can edit your page and change the position of specific block to make it more visible. What's more, Page Builder offers you ready-to-use page blocks that can help to create content tailored to each individual customer:
+
+A. **Default** blocks:
+
+- Dynamic targeting - embeds recommended items based on the Segment the user belongs to.
+- Personalized - displays a list of Content items/products that are recommended to end users when specific scenarios are triggered.
+- Targeting - embeds an Content item based on the Segment the user belongs to.
```
Comment (dabrt) [inline docs/content_management/pages/page_builder_guide.md:201]:
```suggestion
- Targeting - embeds a Content item based on the Segment the user belongs to.
```

**Review by dabrt — COMMENTED** [review]
Please see my comments

File: docs/content_management/pages/page_builder_guide.md [hunk 22]
```diff
@@ -0,0 +1,213 @@
...
+- Configuration - allows you to access configuration window
+- Duplicate - duplicates existing block by creating its copy
+- Refresh - refreshes preview of the block
+- Delete - deletes existing block
+
+#### Schedule content
+
+Page Builder comes with a Scheduler - is a tool that allows you to schedule content publication.
+
+You can schedule to be published, or hidden in Page Builder in two ways with:
```
[the remark this answers is not in the document]
Comment (adriendupuis) [reply docs/content_management/pages/page_builder_guide.md:163]: Content are already "published" from a Repository point of view. They just appear then disappear from the block.
```suggestion
You can schedule content to be revealed, or hidden in Page Builder in two ways with:
```

File: docs/content_management/pages/page_builder_guide.md [hunk 23]
```diff
@@ -0,0 +1,213 @@
...
+
+[Page](pages.md) is a block-based type of content. You can create it and modify with a visual drag-and-drop editor - Page Builder.
+Page is divided into zones into which you can drop various dynamic blocks.
+By editing pages you can customize the layout and content of your website.
+
+### Create page
+
+To create a new page:
+
+1\. In the upper toolbar click **Content**.
```
[the remark this answers is not in the document]
Comment (julitafalcondusza) [reply docs/content_management/pages/page_builder_guide.md:18]: 1\. - fixed. 
My idea was: Page -> Page Builder. It makes sense to me.

File: docs/content_management/pages/page_builder_guide.md [hunk 24]
```diff
@@ -0,0 +1,213 @@
...
+
+You can edit any existing page by using Page Builder. To do it, in the Back Office go to **Content** and select **Content structure**.
+Then, from the Content Tree choose the page that you want to edit and click **Edit**.
+
+## What is Page Builder
+
+Page Builder is a visual tool that allows you to create and edit any page in [[= product_name =]].
+It is more than managing: it is about building pages, creating customized content and fully-targeted landing pages.
+Creating pages in Page Builder involves composing content using ready-to-use elements - blocks, properly configured and customized.
+It is also important to choose a layout - it determines the arrangement of drop zones that contain content elements.
```
[the remark this answers is not in the document]
Comment (julitafalcondusza) [reply docs/content_management/pages/page_builder_guide.md:40]: It's just general description. Page Builder is mostly about adding blocks, that's why I described it in the beginning of this section. 
Looking at detailed sections, the order is correct - "Choose layout", then "Add blocks" and "Work with blocks".

File: docs/content_management/pages/page_builder_guide.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (julitafalcondusza) [reply docs/content_management/pages/page_builder_guide.md:107]: I'd rather keep current link.

File: docs/content_management/pages/page_builder_guide.md [hunk 25]
```diff
@@ -0,0 +1,213 @@
...
+
+![Elements toolbar](elements_toolbar.png)
+
+- Structure view - shows a structure of the page, including its division into zones and the blocks that it contains. It follows the behavior of the content tree. Structure view has ability to reorder blocks using drag and drop.
+
+![Structure view](structure_view.png)
+
+#### Create page in Page Builder
+
+To open the Page Builder, click **Site** and select a website from the list that appears next to the List icon.
```
[the remark this answers is not in the document]
Comment (julitafalcondusza) [reply docs/content_management/pages/page_builder_guide.md:102]: Tested again, I deleted all the section as it's not clear.

File: docs/content_management/pages/page_builder_guide.md (hunk 11 above)
Comment (julitafalcondusza) [reply docs/content_management/pages/page_builder_guide.md:155]: With settings :) info added

File: docs/content_management/img/create_page.png
[the remark this answers is not in the document]
Comment (julitafalcondusza) [reply docs/content_management/img/create_page.png:1]: Good point! Updated

**Review by MagdalenaZuba — APPROVED** [review]

**Review by juskora — APPROVED** [review]

**Review by adriendupuis — APPROVED** [review]

**Pull request merged**